package-canlib.bash

```diff
@@ -1,56 +1,58 @@
 #!/bin/bash
-
-# Exit if any command fails
 set -e
 
-PWD=$(pwd)
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 OS_VER=$(lsb_release -cs)
+SDK_COMMIT="$1"
+VER_SUFFIX="$2"
+
+if [ -z "$SDK_COMMIT" ]; then
+  SDK_COMMIT=master
+fi
 
 echo ""
 echo "Remember to modify the changelog in debian-canlib to include the current release notes."
 echo "If you need to modify it now, hit [CTRL+C]. Otherwise hit [ENTER] to continue."
 read STUFF
 
+cd "$SCRIPT_DIR"
+
 # Check for required files/folders
-if [ ! -e "$PWD/canlib/debian-canlib" ]; then
+if [ ! -e "canlib/debian-canlib" ]; then
   echo ""
   echo "debian-canlib directory not found in canlib folder. Exiting..." 1>&2
   exit -1
 fi
 
-if [ ! -e "$PWD/canlib/Makefile-canlib" ]; then
+if [ ! -e "canlib/Makefile-canlib" ]; then
   echo ""
   echo "Makefile-canlib not found in canlib folder. Exiting..." 1>&2
   exit -1
 fi
 
-# Delete BUILD directory if it exists
-if [ -d "$PWD/BUILD" ]; then
-  rm -r BUILD/
-fi
-
+rm -rf BUILD/
```

Comment:
This will return an error if the folder doesn't exist, which is why I wrapped it in the `if` statement.

Reply:
The `-f` will suppress errors due to non-existent things.